front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -43,6 +43,7 @@ import {
 import { Locale } from 'typings';
 
 type ContentBuilderTopBarProps = {
+  isPendingState: boolean;
```

Comment:
Non-blocking. We could rename this to `hasPendingState`. I'd also consider making it optional or false by default since it handling a specific kind exception.